doc/developer/webservices.rst

```diff
@@ -279,8 +279,10 @@ Connected JSON result
    {
        "username": "<username>",
        "is_password_changed": "True"/"False", # If false the user should change his password
-       "role_name": "<role_name>",
-       "role_id": <role_id>
+       "roles": [{
+           "name": "<role_name>",
+           "id": <role_id>
+       }, ...],
        "functionality": {
            "<functionnality_name>": ["functionnality_value"],
```

Comment:
functionality_name and functionality_value
(I know, it's not really part of this PR...)

Reply:
Fixed :-)